pkg/apis/clickhouse.altinity.com/v1/type_chi.go

```diff
@@ -25,28 +25,26 @@ import (
 	"gopkg.in/yaml.v3"
 
 	"github.com/altinity/clickhouse-operator/pkg/util"
-	"github.com/altinity/clickhouse-operator/pkg/version"
 )
 
 // FillStatus fills .Status
 func (chi *ClickHouseInstallation) FillStatus(endpoint string, pods, fqdns []string, ip string) {
-	chi.EnsureStatus().CHOpVersion = version.Version
```

Comment:
version is always statically available so these 3 field setter calls were pushed within the new `Fill` function (and therefore absent from the passed params)